identifiers/country-us/us_territory_government.csv

```diff
@@ -0,0 +1,8 @@
+id,name
+ocd-division/country:us/territory:mp/region:boe_1,Northern Mariana Islands Board of Education District 1
```

Comment:
Why are these region:boe_1 instead of board_of_education:1 ? (`vi` uses the `board_of_education` type.)